### PR TITLE
Fixed phpdoc

### PR DESCRIPTION
**Pull request by core23**

### CODE REVIEW

File: Pager/BasePager.php [hunk 1]
```diff
@@ -20,20 +20,69 @@
...
+     * @var bool
+     */
     protected $maxRecordLimit = false;
+
+    /**
+     * @var int
+     */
     protected $maxPageLinks = 0;
 
-    // used by iterator interface
```
Comment (Soullivaneuh) [inline Pager/BasePager.php]: This should be kept IMO.

File: Pager/Doctrine/Pager.php [hunk 2]
```diff
@@ -21,10 +21,13 @@
  */
 class Pager extends BasePager
 {
+    /**
+     * @deprecated This property is deprecated since version 2.2 and will be removed in 3.0.
```
Comment (Soullivaneuh) [inline Pager/Doctrine/Pager.php]: Well, no stable version yet for this bundle.

This may be removed directly.

ping @rande

Comment (rande) [inline Pager/Doctrine/Pager.php]: +1

**Comment by core23** [comment]
Should be fine now

**Pull request merged**